.changeset/real-pots-drive.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-docs/gatsby-theme-api-docs': minor
+'@commercetools-api-specs/test': patch
+'@commercetools-website/api-docs-smoke-test': patch
```

Comment:
neither tests nor the test websites are released artifacts.  Please remove them from the changeset.